### ISSUE TITLE
Conflict with CommunityToolkit.WinUI.Collection

### ISSUE THREAD

**Issue by ghost1372** [w-ahmad/WinUI.TableView#52]
i used AdvancedCollectionView in my project (latest version), then i installed your TableView library, and i get conflict errors:

`The type 'AdvancedCollectionView' exists in both 'CommunityToolkit.WinUI.Collections, Version=8.2.0.0, Culture=neutral, PublicKeyToken=null' and 'WinUI.TableView, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'`

I need to use AOT in my Apps and it seems that the bugs related to AOT have been fixed in the latest versions.
But it seems that AdvancedCollectionView is hardcoded in your library and we are forced to use your AdvancedCollectionView and we cannot use its newer versions.
so please fix it

**Comment by mcka-dev** [comment]
The bug is present in the latest version:

`The type 'IAdvancedCollectionView' exists in both 'CommunityToolkit.WinUI.Collections, Version=8.1.0.0, Culture=neutral, PublicKeyToken=null' and 'WinUI.TableView, Version=1.2.2.0, Culture=neutral, PublicKeyToken=null'`

**Comment by w-ahmad** [comment]
would it work if I rename the `AdvancedCollectionView` and `ITableViewCollectionView` to something else? I am asking because this code is from Community Toolkit.

**Comment by ghost1372** [comment]
> is it fine if I rename the `AdvancedCollectionView` and `ITableViewCollectionView` to something else? I am asking because this code is from Community Toolkit. 

No, it is better to refrence communitytoolkit collection nuget package and remove this internal classes

**Comment by w-ahmad** [comment]
I wish I could, but the problem is I must modify it to my needs that doesn't fit for the communitytoolkit.
